packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -146,7 +146,7 @@ const VideoEffectsPaneTrampoline = (
   /* @conditional-compile-remove(video-background-effects) */
   return (
     <Stack horizontalAlign="center">
-      {videoEffectError && (
+      {videoEffectError && isCameraOn && (
```

Comment:
is this correct? I thought bug fixes should be coming int from main into the release branch and so we shouldnt be seeing any new code when we merge back into main?

Reply:
Is it in main branch already or not? Which fix is this coming from? Wondering why this is introduced here